packages/react/src/oxygen-ui-react-auth-components/SignInAlert/sign-in-alert.scss

```diff
@@ -0,0 +1,3 @@
+.OxygenSignInAlert {
```

Comment:
Let's add the license headers

Reply:
added with 06f7ef89e7d1013b7029e5ee47bd6da2fd6e07e1